app/utils/redis.py

```diff
@@ -0,0 +1,35 @@
+import redis
+
+from app.core.config import Settings
+
+
+def connect(settings: Settings) -> redis.Redis | bool:
+    try:
+        redis_client = redis.Redis(host=settings.REDIS_HOST, port=settings.REDIS_PORT)
+        redis_client.ping()  # Test the connection
+    except redis.exceptions.ConnectionError as e:
+        print("Redis connection error: ", e)
+        print("Check the Redis configuration  or the Redis server")
```

Comment:
Could you use `hyperion.error` logger?

Reply:
I think that using the the logger in this file wouldn't make much sense. Then how about I to just delete the exception handling in this file and put it in the dependencies ?

Reply:
I agree, it makes sense